src/main/scala/rocket/PTW.scala

```diff
@@ -26,11 +26,15 @@ class PTWReq(implicit p: Parameters) extends CoreBundle()(p) {
 class PTWResp(implicit p: Parameters) extends CoreBundle()(p) {
   val ae_ptw = Bool()
   val ae_final = Bool()
+  /**page fault  */
   val pf = Bool()
+  /**guest page fault  */
   val gf = Bool()
+  //hypervisor wrx
```

Comment:
You need to read this: https://docs.scala-lang.org/overviews/scaladoc/for-library-authors.html